packages/host/app/components/operator-mode/profile-info-popover.gts

```diff
@@ -192,29 +208,32 @@ export default class ProfileInfoPopover extends Component<ProfileInfoPopoverSign
     return await this.realmServer.fetchCreditInfo();
   });
 
+  private get isLoading() {
+    return this.fetchCreditInfo.isLoading;
+  }
+
   private get plan() {
     return this.fetchCreditInfo.value?.plan;
   }
 
+  private get planName() {
+    return this.plan?.name ?? '-';
```

Comment:
Probably we don't want a case where user does not have a plan

Reply:
Should I hide the credit section in account popover when user does not subscribe to any plan?

Reply:
No, I mean that I think user should be subscribed to one of the plans in order to use the app. CS-7396 will ensure user starts with a free plan, and then they can switch between plans. I don't see a case here where user is not on a plan

Reply:
Does that mean in the endpoint we can throw NotFound error if user has no subscription, instead of returning `{ data: null, included: null }` ?

Reply:
I think it's ok if you return the user data (matrixUserId,..) without subscription data included (or empty subscriptions array)

Reply:
@lukemelia  we’d appreciate your input.

Reply:
A user should not be able to access Boxel operator mode unless they have a plan. Users who do not should be shown a message explaining this with a button to sign up for the free plan.